cypress/integration/brokerIsAbleToUpdateStatusOfInquiry.feature.js

```diff
@@ -0,0 +1,71 @@
+describe('Broker is able to update status of inquiry', () => {
+  beforeEach(() => {
+    cy.intercept(
+      'GET',
+      'https://flex-coast-development.herokuapp.com/api/inquiries',
+      {
+        fixture: 'listOfInquiries.json',
+      }
+    )
+    cy.visit('/')
+    cy.get('[data-cy=inquiry]').first().click()
+  })
+  describe('Successfully', () => {
+    beforeEach(() => {
+      cy.intercept(
+        'PUT',
+        'https://flex-coast-development.herokuapp.com/api/inquiries/**',
+        {
+          message: 'The inquiry status has been updated',
+        }
+      )
+    })
+    it('is expected to update the status to started', () => {
+      cy.get('[data-cy=inquiry-collapsible-cell]').within(() => {
+        cy.fixture('listOfInquiries.json').then((fixture) => {
+          fixture.inquiries[0].inquiry_status = 'started'
```

Comment:
is this like just for picking the [0] kind of first one in fixture?

Reply:
Yeah it's for targeting the first inquiry in the `inquiries` list and modifying that :)